apache/datadog_checks/apache/apache.py

```diff
@@ -123,12 +123,14 @@ def check(self, instance):
             # Can't get it from the mod_status output, try to get it from the server header even though
             # it may not be exposed with some configurations.
             server_version = r.headers.get("Server")
-            if server_version:
+            if server_version.startswith('Apache'):
```

Comment:
I'm not sure this `.startswith()` check is required here, since if it doesn't start with `'Apache'` then the pattern won't match, and we'll log it (which we won't in this case). Also we're not doing this check in the other case on L85 (when using the `ServerVersion` metric).